src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
I like removing `.originalEvent`. If we can use `delete` without any browsers spitting up, I'd like that best.